device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/phy2_config_1.json

```diff
@@ -0,0 +1,101 @@
+{
+  "lanes": [
```

Comment:
it is unclear how the lanes are mapped to the ports, where do we get those information?

Reply:
The section of lanes is not used, and the data put here is to pass the parser checking.  The related info is obtained from gearbox_100G_PAM4.xml, the file only applied to credo driver.